m4opt/models/extinction/__init__.py

```diff
@@ -0,0 +1,15 @@
+"""
+Extinction models: models for light attentuation due to Earth's atmosphere.
+Includes Airmass Object for airmass calculation.
+
+Extinction models are defined at the observatory location and with an
+extinction table. They take the sky coordinate of the target as input,
+and return an attentuation factor (dimensionless) as output.
+
+Airmass models are defined at the observatory location and return the airmass
+for a given target in the sky.
+"""
+
+from .airmass import Airmass, AtmosphericExtinction
+
+__all__ = ('Airmass', 'AtmosphericExtinction',)
```

Comment:
```suggestion
from .airmass import *
```
this goes with the previous suggestion adding an `__all__ to `airmass.py`. This defers the question of "what API is actually exposed" to the `airmass.py`, which I think is cleaner because you can read it from the file where it's implemented.